src/website/src/app/documentation/demos/modal/modal.demo.html

```diff
@@ -206,6 +206,21 @@ <h3 id="examples">Code &amp; Examples</h3>
                 </div>
             </div>
 
+            <div class="alert alert-warning bump-down">
+                <div class="alert-items">
+                    <div class="alert-item static">
+                        <div class="alert-icon-wrapper">
+                            <clr-icon class="alert-icon" shape="exclamation-triangle"></clr-icon>
+                        </div>
+                        <span class="alert-text">
+                            In the examples below <code class="clr-code">aria-hidden</code> is there to point where you have to
+                            place it. But its value has to be changed depending on the state the modal is. This is already done
```

Comment:
❤️

Reply:
![you](https://media.giphy.com/media/ALQnCUBuggPxS/giphy.gif)